src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculate from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculate />
     </div>
   );
 }
```

Comment:
- [ ] Please use class-based component instead function-based component as required in the project requirements.

### See the image of the requirements

![image](https://user-images.githubusercontent.com/83803414/176451685-a9a8e4b8-255a-4fbb-8c5b-ceecc6a269f2.png)